workloads/router-perf-v2/README.md

```diff
@@ -39,16 +52,13 @@ It's possible to tune the default configuration through environment variables. T
 | RUNTIME				| Workload duration in seconds | `120` |
 | TERMINATIONS  		| List of HTTP terminations to test | `http edge passthrough reencrypt mix` |
 | URL_PATH              | URL path to use in the benchmark | `/1024.html` |
-| NUMBER_OF_ROUTES		| Number of routes of each termination to test. (the mix scenario test all of them together) | `100` |
```

Comment:
out of curiosity, why are these parameters being removed? Do we no longer expect user to be able to configure these items? I feel like leaving it configurable maybe still valuable for us? Thoughts @mffiedler ?